server.js

```diff
@@ -0,0 +1,35 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+const multer = require("multer");
+const cors = require("cors");
+
+var forms = multer();
```

Comment:
Don't ever use `var` it is totally outdated

Replace all occurrences of `var `with `const `(or `let`)